### ISSUE TITLE
Test failures

### ISSUE THREAD

**Issue by nk9** [Thom1729/Sublime-JS-Custom#139]
When I run Build With: Syntax Tests - All Syntaxes after installing JS Custom, I get a bunch of failures in your project. This happens on both Sublime 4143 (the latest stable release) and 4149 (the latest dev release).

```
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:5: [meta.selector variable.language.css] does not match scope [source.js.css variable.language.css.styled-components]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:6: [meta.selector punctuation.definition.pseudo-class] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:7: [meta.selector entity.other.pseudo-class] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:8: [meta.selector entity.other.pseudo-class] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:9: [meta.selector entity.other.pseudo-class] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:10: [meta.selector entity.other.pseudo-class] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:11: [meta.selector entity.other.pseudo-class] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:20:13: [meta.property-list meta.block punctuation.section.block.begin] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:7: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:8: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:9: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:10: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:11: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:12: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:13: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:14: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:15: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:16: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:17: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:18: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:19: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:20: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:21: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:22: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:23: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:24: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:25: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:26: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:27: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:28: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:29: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:30: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:31: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:32: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:33: [meta.property-list meta.block] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:7: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:8: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:9: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:10: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:11: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:12: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:13: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:14: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:15: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:16: [meta.property-name support.type.property-name] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:17: [punctuation.separator.key-value] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:19: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:20: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:21: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:22: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:23: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:24: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:25: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:26: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:27: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:28: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:29: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:30: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:31: [meta.property-value support.constant.color.w3c.extended] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:25:32: [punctuation.terminator.rule] does not match scope [source.js.css]
Packages/JSCustom/styled_components/syntax_test_styled_components.styled-components:31:5: [meta.property-list meta.block punctuation.section.block.end] does not match scope [source.js.css]
```

**Comment by Thom1729** [comment]
Probably the same thing as https://github.com/babel/babel-sublime/issues/427.

My best guess is that the core CSS syntax was updated and the Styled Components syntax, which extends it, broke as a result.

I'm on vacation but should have a chance to take a look tonight or tomorrow.

**Comment by Thom1729** [comment]
Yep, the CSS syntax was updated and the context names changed. I've hacked it so that it should work with both the old and new CSS syntaxes. v4.2.3 should fix it.

**Comment by Thom1729** [comment]
Fixed in v4.2.3.
